src/bundle/ui-dev/src/modules/universal-discovery/components/content-create-widget/content.create.widget.js

```diff
@@ -115,9 +116,11 @@ const ContentCreateWidget = () => {
                     <div className="ibexa-extra-actions__section-header">{selectLanguageLabel}</div>
                     <div class="ibexa-extra-actions__section-content">
                         <Dropdown
+                            dropdownListRef={DropdownPortalRefContext}
```

Comment:
It is not a ref, but a context of a ref
```suggestion
                            portalRefContext={DropdownPortalRefContext}
```